calico_kubernetes/calico_kubernetes.py

```diff
@@ -254,9 +276,9 @@ def _get_container_info(self, container_name):
             info = self._docker_client.inspect_container(container_name)
         except APIError as e:
             if e.response.status_code == 404:
-                print("Error: Container %s was not found. Exiting." % container_name)
+                logger.errror("Container %s was not found. Exiting." % container_name)
```

Comment:
When using python logging, you can ditch the "%" operator to do string formatting.

Its more efficient to use the following syntax, because then the string formatting only occurs if the log is emitted (for example, debug logs won't get emitted when logging is set to INFO)

```
logger.error("Container %s was not found", container_name)
```